_data/sites.json

```diff
@@ -1520,6 +1520,17 @@
         ]
     },
 
+    {
+        "name": "Bol.com",
+        "url": "https://www.bol.com/nl/rnwy/account/privacy/delete",
+        "difficulty": "hard",
+        "notes": "Need to request deletion on a page within the account settings. Confirm after via link in email, which often doesn't work, because you need to be logged in to the account for the confirmation link to be working. This is not clear.",
+        "email": "dpo@aholddelhaize.com",
```

Comment:
```suggestion
```

Reply:
This is the correct email.

Reply:
Ah, after further explaining that the link might not work, I agree with keeping the e-mail